source/change-streams/change-streams.rst

```diff
@@ -398,14 +446,19 @@ startAtOperationTime
 
 The server expects ``startAtOperationTime`` as a BSON Timestamp. Drivers MUST allow users to specify a ``startAtOperationTime`` option in the ``watch`` helpers. They MUST allow users to specify this value as a raw ``Timestamp``.
 
-``startAtOperationTime`` and ``resumeAfter`` are mutually exclusive; if both ``startAtOperationTime`` and ``resumeAfter`` are set, the server will return an error. Drivers MUST NOT throw a custom error, and MUST defer to the server error.
+``startAtOperationTime``, ``resumeAfter``, and ``startAfter`` are all mutually exclusive; if any two are set, the server will return an error. Drivers MUST NOT throw a custom error, and MUST defer to the server error.
 
-If neither ``startAtOperationTime`` nor ``resumeAfter`` are specified, and the max wire version is >= ``7``, and the initial ``aggregate`` command does not return a resumeToken (indicating no results), the ``ChangeStream`` MUST save the ``operationTime`` from the initial ``aggregate`` command when it returns.
+The ``ChangeStream`` MUST save the ``operationTime`` from the initial ``aggregate`` response when the following critera are met:
+
+- None of ``startAtOperationTime``,  ``resumeAfter``, ``startAfter`` were specified in the ``ChangeStreamOptions``.
+- The max wire version is >= ``7``.
+- The initial ``aggregate`` response did not include a ``documentResumeToken`` (indicating no results).
+- The initial ``aggregate`` response did not include a ``postBatchResumeToken``.
 
 resumeAfter
```

Comment:
As a side-note, I think we should add a `startAfter` section (since we have separate sections for `resumeAfter` and `startAtOperationTime`), which could possibly capture the info in [this comment](https://jira.mongodb.org/browse/SERVER-35030?focusedCommentId=1962023&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-1962023). If you agree, I'll create a follow-up spec ticket after this is merged.